rev-connect-api/src/main/java/com/rev_connect_api/config/WebConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.rev_connect_api.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**") //
+//                .allowedOrigins("http://localhost:3000") //
+                .allowedOrigins("*") //
+                .allowedMethods("GET", "POST", "PUT", "DELETE", "OPTIONS") //
+                .allowedHeaders("*"); //
+//                .allowCredentials(true); //
```

Comment:
We probably want to specify the cross-origin URL. The frontend can proxy HTTP requests to avoid CORS conflicts so we may opt to go with that route so the backend is ambiguous of the front end